app/src/main/java/com/example/peterchu/watplanner/home/HomeFragment.java

```diff
@@ -128,6 +131,27 @@ public void onStart() {
         homePresenter.start();
     }
 
+    public void showExportCalendarDialog() {
+        AlertDialog.Builder builder = new AlertDialog.Builder(getContext());
+        builder.setTitle("Export added courses to your schedule.");
+
+        builder.setPositiveButton("Export", new DialogInterface.OnClickListener() {
+            @Override
+            public void onClick(DialogInterface dialog, int which) {
+                homePresenter.onExportCoursesToCalendar(new ArrayList<CourseComponent>());
```

Comment:
this needs to be replaced with a list of course components that the user has selected, or the configuration of course components for courses that the SAT solver came up with